a3p-integration/proposals/a:upgrade-next/core-eval-support.js

```diff
@@ -0,0 +1,53 @@
+// @ts-check
```

Comment:
hm. another copy of this file? we can't get the relevant stuff from the synthetic chain package?

Reply:
If there's a way, I'm open to it. I just extracted what you had in https://github.com/Agoric/agoric-sdk/pull/8786. I'm open to alternatives!

Reply:
I think I found a way. PTAL